tests/testthat/test-run_stress_test.R

```diff
@@ -0,0 +1,23 @@
+test_that("is sensitive to `asset_type`, and defaults to bonds", {
+  local_dev_env()
+  expect_equal(run_stress_test("equity"), "equity")
+})
+
+test_that("with invalid `asset_type` errors gracefully", {
+  expect_error(run_stress_test("invalid"), "should be.*bonds.*equity.*loans")
+})
+
+test_that("if `asset_type` is longer than 1 uses default with a warning", {
+  local_dev_env()
+  expect_snapshot(out <- run_stress_test())
+  expect_equal(out, "bonds")
+})
+
+test_that("returns the first argument invisibly", {
+  skip_on_ci()
+  skip_on_cran()
+
+  readRenviron(here::here(".Renviron"))
```

Comment:
1. can u please elaborate why readEnviron needs to be put here? it depends on using an .renviron on project level, which neither @jacobvjk  nor me currently do

Reply:
It's  make the environmental variables available inside the test. The testing environment is independent and what works interactively may not work in a test. 

I didn't quite understand how you set the environmental variables. I could try re-read that but easiest to test is to not need environmental variables.

Reply:
i simply use a global .renv, which is thus not readable via here::here.

Omitting env vars would force us to add two input args to determine the data sources locations and even then we would have to interact with the file system which I would prefer the test not to do. I am afraid with this limiting factor it is not possible to make a clean test unless we:

- abstract away bits of the code that interact, and have to interact, with the file system and test the remaining code in integration, using synthetic input data. 
- or setup a temp env with all needed inputs, again using synthetic input data
- While that is among the first things on the prio list after the deadline it is not realistic to implement before, thus I would suggest to either remove this test or disable it per default.